### DISCUSSION TITLE
How to prevent side UI from being cropped off on small devices.

### DISCUSSION THREAD

**Discussion by iJoel-i** [Stellarium/stellarium#3383] [Q&A] [answered]
A few of us are using stellarium for a class and the side UI gets cut off on some devices. Any good way to fix this? (I mean this UI) Not cutoff much for me, but is for others
![image](https://github.com/Stellarium/stellarium/assets/143626706/34a6d04d-b511-4a26-8633-fb16fa624dcf)

**Comment by 10110111** [comment]
Try reducing font size, or switching your screen to landscape orientation. Desktop Stellarium is not really intended for low-res screens, so I don't think you'll find any better solution.

**Comment by alex-w** [comment, marked as answer]
F2 -> Information

**Reply by iJoel-i** [reply]
Perfect. Going to tools and can change font size. Thank you for the help.

**Comment by gzotti** [comment]
On narrow screens, you can also think whether all info bits are relevant for you, and configure some of them away. Galactic/supergalactic coordinates? Both ecliptical, both equatorial?
